src/api/EscargotPublic.h

```diff
@@ -1777,6 +1777,18 @@ class ESCARGOT_EXPORT PlatformRef {
 public:
     virtual ~PlatformRef() {}
 
+    // ArrayBuffer
+    // client must returns zero-filled memory
+    virtual void* onArrayBufferObjectDataBufferMalloc(size_t sizeInByte)
+    {
+        return calloc(sizeInByte, 1);
+    }
```

Comment:
```Malloc``` is used in the name of this method, but ```calloc``` is called actually.

Reply:
IMO malloc is better.
calloc means contiguous alloc. but I want just zero-filled memory